plugins/lookup/bitwarden_secrets_manager.py

```diff
@@ -84,11 +85,28 @@ class BitwardenSecretsManagerException(AnsibleLookupError):
 class BitwardenSecretsManager(object):
     def __init__(self, path='bws'):
         self._cli_path = path
+        self._max_retries = 3
+        self._retry_delay = 1
 
     @property
     def cli_path(self):
         return self._cli_path
 
+    def _run_with_retry(self, args, stdin=None, retries=0):
+        if retries > self._max_retries:
```

Comment:
This is run after the last (very long) delay. How about moving it into the `if "Too many ...` below, and changing it to `if retries >= self._max_retries`? Or simply add ` ... and retries >= self._max_retries` to the `if` conditions below?

Reply:
[ce75699](https://github.com/ansible-collections/community.general/pull/8238/commits/ce756996a72b9f422e1f0630dddc98a6b1e6ba60) Should avoid that extra execution now